modules/ztp-preparing-the-hub-cluster-for-ztp.adoc

```diff
@@ -41,3 +41,38 @@ You can configure the hub cluster with a set of ArgoCD applications that generat
 
 [start=2]
 include::snippets/ztp-patch-argocd-hub-cluster.adoc[]
+
+
+. Verify that the `PrunePropagationPolicy=background` policy is set in the `Application` resource by running the following command:
++
+--
+[source,terminal]
+----
+$ oc -n openshift-gitops get applications.argoproj.io  \
+clusters -o jsonpath='{.spec.syncPolicy.syncOptions}' |jq
+----
+
+.Example output for existing policy
+[source,terminal]
+----
+[
+  "CreateNamespace=true",
+  "PrunePropagationPolicy=background",
+  "RespectIgnoreDifferences=true"
+]
+----
+--
+
+.. If the policy does not exist, set the `PrunePropagationPolicy` field to `background` in the `Application` resource:
```

Comment:
"f the syncOption does not exist, or is configured with foreground value, set the `PrunePropagationPolicy` field to `background` in the `Application` resource: